tests/attested_tls/common/settings.c

```diff
@@ -0,0 +1,24 @@
+#include "settings.h"
```

Comment:
You can remove settings.c file. 
In the referred files use "#include <stdlib.h> and getenv() function directly.